static/js/util/currency.js

```diff
@@ -37,7 +37,7 @@ const codesToOptions = R.compose(
 
 export const currencyOptions = codesToOptions(cc.codes());
 
-const codeToCountryName = code => iso3166.country(code).name || '';
+export const codeToCountryName = code => iso3166.country(code).name || '';
```

Comment:
functions that are not exported have their types inferred, but for efficient type-checking across modules flow enforces that all exported functions have to have their types annotated.